tgui/packages/tgui/interfaces/CargoImportConsole.js

```diff
@@ -147,49 +117,26 @@ export const CargoImportConsole = (props, context) => {
                               </Box>
                             </Stack.Item>
                             <Stack.Item>{item.description}</Stack.Item>
-                            {!!item.cant_purchase && (
-                              <Stack.Item textColor={'red'}>
-                                {
-                                  'Company interest needs to be higher to buy this item!'
-                                }
-                              </Stack.Item>
-                            )}
                             {!!cant_buy_restricted && !!item.restricted && (
                               <Stack.Item textColor={'red'}>
                                 {
                                   'You cannot buy a restricted item from this console!'
                                 }
                               </Stack.Item>
                             )}
-                            <Stack.Item
-                              textColor={item.quantity <= 0 ? 'red' : 'green'}>
-                              {'Quantity Remaining: ' + item.quantity}
-                            </Stack.Item>
                             <Stack.Item
                               textColor={
                                 item.cost > budget_points ? 'red' : 'green'
                               }>
                               {'Cost: ' + item.cost}
                             </Stack.Item>
-                            {!!item.buyable_ammo && (
-                              <Stack.Item
-                                textColor={
-                                  item.magazine_cost > budget_points
-                                    ? 'red'
-                                    : 'green'
-                                }>
-                                {'Ammo Cost: ' + item.magazine_cost}
-                              </Stack.Item>
-                            )}
```

Comment:
So you can't buy the ammo anymore?

Reply:
Ammo is now listed alongside the weapons rather than in the menu. This allows for future plans especially but mainly lets me easier sell multiple different types of magazines for the same weapon without having to duplicate all of the code for buying stuff. If that doesn't make sense trust me the magazine buying thing made less sense.